include/pops/treatments.hpp

```diff
@@ -175,29 +176,40 @@ class SimpleTreatment : public BaseTreatment<IntegerRaster, FloatRaster>
         IntegerRaster& infected,
         std::vector<IntegerRaster>& exposed,
         IntegerRaster& susceptible,
-        IntegerRaster&,
+        IntegerRaster& resistant,
+        IntegerRaster& total_hosts,
         const std::vector<std::vector<int>>& suitable_cells) override
     {
         for (auto indices : suitable_cells) {
             int i = indices[0];
             int j = indices[1];
+            int exposed_changed = 0;
+            int infected_changed = 0;
+            int susceptible_changed = 0;
```

Comment:
Please name these something like `new_infected, new_susceptible`, changed is little misleading. For `exposed_changed`, maybe `new_exposed_total` to be really clear.